lib/vtls/openssl.c

```diff
@@ -3386,7 +3357,7 @@ static CURLcode ossl_connect_step2(struct Curl_easy *data,
 
     /* Informational message */
     infof(data, "SSL connection using %s / %s\n",
-          get_ssl_version_txt(backend->handle),
+          SSL_get_version(backend->handle),
```

Comment:
OpenSSL supports this all the way back to 0.9.8 so this seems like a neat simplification.